packages/amplication-data-service-generator/src/util/message-broker.ts

```diff
@@ -0,0 +1,37 @@
+import { DSGResourceData, ServiceTopics } from "@amplication/code-gen-types";
+import { EnumResourceType } from "../models";
+
+export function fromServiceTopicsToTopicsWithName(
+  serviceTopicsList: ServiceTopics[],
+  messageBrokers: DSGResourceData[]
```

Comment:
I think you can send the entire dsgResourceData object (or event the context object) to the function, and then extract the messageBrokers and topics instead of asking the calling function to do it for you

Reply:
I don't think like you, we need to send only the relevant information to the function